assets/page/SettingsAccount.svelte

```diff
@@ -76,6 +79,10 @@ function updateUserFromForm(e) {
     <Checkbox name="notifications" bind:checked="{wantNotifications}">
       <span slot="label">{l('Enable notifications')}</span>
     </Checkbox>
+    
+    <Checkbox name="statuses" bind:checked="{ignoreStatuses}">
+      <span slot="label">{l('Ignore statuses')}</span>
```

Comment:
Can we make that easier to understand? Like "Ignore join/part messages" or something like that?

Reply:
Fixed